### ISSUE TITLE
"make istioctl" no longer works

### ISSUE THREAD

**Issue by esnible** [istio/istio#15192]
"make istioctl" no longer works.

```
$ make istioctl
make: Nothing to be done for `istioctl'.
```

but the code has changed; I added a typo

```
$ bin/gobuild.sh ${ISTIO_OUT}/istioctl ./istioctl/cmd/istioctl
# istio.io/istio/istioctl/cmd
istioctl/cmd/kubeinject.go:311:40: invalid character U+005C '\'
```

This seems to have been caused by https://github.com/esnible/istio/commit/3ef921b984bcb5f8753c096613c712c4e995c6c2#diff-b67911656ef5d18c4ae36cb6741b7965

I am on OSX.

**Comment by howardjohn** [comment]
Fails on linux too

**Comment by nmittler** [comment]
Sorry about the breakage. Fix in #15196
